CentralHub.Api/Services/LocalizationTargetService.cs

```diff
@@ -0,0 +1,27 @@
+using CentralHub.Api.Model;
+
+namespace CentralHub.Api.Services;
+
+class LocalizationTargetService
+{
+    private static Dictionary<int, List<Measurement>> Measurements = new Dictionary<int, List<Measurement>>();
+
+    public static async Task AddMeasurementsAsync(int id, List<Measurement> measurements, CancellationToken token)
+    {
+        if (Measurements.ContainsKey(id))
+        {
+            Measurements[id].AddRange(measurements);
+        }
+        else
+        {
+            Measurements.Add(id, measurements);
+        }
+        await Task.CompletedTask;
+    }
+
+    public static async Task<List<Measurement>> GetMeasurementsForId(int id, CancellationToken token)
+    {
+        return await Task.FromResult(Measurements.GetValueOrDefault(id, new List<Measurement>()));
+    }
+
+}
```

Comment:
Make these non-static and use dependency injection as a singleton

Reply:
Also this is not thread safe. It should be.

Reply:
It should also implement ILocalizationTargetService so we can mock it in unit tests